test/specs/runApexTests.e2e.ts

```diff
@@ -0,0 +1,521 @@
+  //   expect(outputPanelText).toContain('=== Test Summary');
+  //   expect(outputPanelText).toContain('TEST NAME');
+  //   expect(outputPanelText).toContain('ExampleApexClass1Test.validateSayHello');
+  //   expect(outputPanelText).toContain('ended SFDX: Run Apex Tests');
+  // });
+
+  // step('Run Single Test via Apex Class', async () => {
+  //   const workbench = await browser.getWorkbench();
+  //   const editorView = workbench.getEditorView();
+  //   let textEditor: TextEditor;
+
+  //   // Open an existing apex test (e.g. BotTest.cls, search for @isTest)
+  //   textEditor = await editorView.openEditor('ExampleApexClass2Test.cls') as TextEditor;
+
+  //   // Click the "Run Test" code lens at the top of one of the test methods
+  //   const codeLens = await textEditor.getCodeLens('Run Test') as CodeLens;
+  //   await (await codeLens.elem).click();
+
+  //   // Wait for the command to execute
+  //   await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Run Apex Tests', 5 * 60);
+  //   await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Run Apex Tests: Listening for streaming state changes...', 5 * 60);
+  //   await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Run Apex Tests: Processing test run', 5 * 60, false);
+
+  //   // Look for the success notification that appears which says, "SFDX: Build Apex Test Suite successfully ran".
+  //   const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Run Apex Tests successfully ran');
+  //   if (successNotificationWasFound !== true) {
+  //     const failureNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Run Apex Tests failed to run');
+  //     if (failureNotificationWasFound === true) {
+  //       expect(successNotificationWasFound).toBe(false);
+  //     }
+  //   } else {
+  //     expect(successNotificationWasFound).toBe(true);
+  //   }
+
+  //   // Verify test results are listed on vscode's Output section
+  //   const outputPanelText = await utilities.attemptToFindOutputPanelText('Apex', '=== Test Results', 10);
+  //   expect(outputPanelText).not.toBeUndefined();
+  //   expect(outputPanelText).toContain('=== Test Summary');
+  //   expect(outputPanelText).toContain('TEST NAME');
+  //   expect(outputPanelText).toContain('ExampleApexClass2Test.validateSayHello');
+  //   expect(outputPanelText).toContain('ended SFDX: Run Apex Tests');
+  // });
+
+  step('Run Tests via Command Palette', async () => {
+    const workbench = await browser.getWorkbench();
+
+    // Run SFDX: Run Apex tests.
+    prompt = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Run Apex Tests', 1);
+
+    // Select the "ExampleApexClass1Test" file
+    prompt.selectQuickPick('ExampleApexClass1Test');
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Run Apex Tests', 5 * 60);
```

Comment:
The unit of the duration parameter that is passed is seconds.  Should it be 5 minutes (like it is her) or 5 seconds?